LanternExtractor/packages.config

```diff
@@ -4,6 +4,7 @@
   <package id="DotNetZip" version="1.10.1" targetFramework="net452" />
   <package id="Fody" version="6.4.0" targetFramework="net48" developmentDependency="true" />
   <package id="GlmSharp" version="0.9.8" targetFramework="net452" />
+  <package id="Ionic.Zip" version="1.9.1.8" targetFramework="net48" />
```

Comment:
Is this needed or used in addition to `DotNetZip`?

Reply:
No idea how that guy got installed, good catch. Probably visual studio being smart about something. Removed